project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/ontology/TibTerm.java

```diff
@@ -0,0 +1,71 @@
+package life.qbic.projectmanagement.infrastructure.ontology;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * <b>Tib Term</b>
+ * <p>
+ * Class representing the TIB Terminology JSON response object for a term.
+ *
+ * @since 1.4.0
+ */
+public class TibTerm {
+
+  @JsonProperty("id")
+  String id;
+  @JsonProperty("iri")
+  String iri;
+  @JsonProperty("short_form")
+  String shortForm;
+  @JsonProperty("obo_id")
+  String oboId;
+  @JsonProperty("label")
+  String label;
+  @JsonProperty("ontology_name")
+  String ontologyName;
+  @JsonProperty("ontology_prefix")
+  String ontologyPrefix;
+  @JsonProperty("description")
+  List<String> description;
```

Comment:
Why is this a list?

Reply:
i dont know, but that is how the response object looks like :D

Reply:
So the API returns a `Property` data type and not a term?

Reply:
You probably need to set `type=class` in the query. Then you get a description which is not a list and is filled correctly